File: PRACTICA DE CAMPO 02-S04/C#/PracticaCampo_02_S04/s04_pc02_suma_n_numeros.cs
int cantidad_numero_ingresado;
int aumento=1;
int sumando=0;
int numero_mayor=0;
int numero_menor=0;
double promedio=0.0;
Console.WriteLine("Indique cantidad de numeros a sumar :");
cantidad_numero_ingresado=int.Parse(Console.ReadLine());

while (aumento<=cantidad_numero_ingresado){
    if (aumento == 1){
        numero_menor=aumento;
    }else if (aumento == cantidad_numero_ingresado){
        numero_mayor=aumento;
    }
    sumando=sumando+aumento;
    aumento++;
}
promedio=(double)sumando/cantidad_numero_ingresado;

Console.WriteLine("Cantidad Números: " + cantidad_numero_ingresado);
Console.WriteLine("Suma Total: " + sumando);
Console.WriteLine("Promedio: " + promedio);
Console.WriteLine("Numero Mayor: " + numero_mayor);
Console.WriteLine("Numero Menor: " + numero_menor);

File: PRACTICA DE CAMPO 02-S04/PYTHON/s04_pc02_suma_n_numeros.py
cantidad_numero_ingresado=int(input("Indique cantidad de de numeros a sumar: "))
aumento=0
sumando=0
numero_mayor=0
numero_menor=0
while True:    
    aumento=aumento+1
    sumando=sumando+aumento
    if aumento==1:
        numero_menor=aumento
    elif aumento==cantidad_numero_ingresado:
        numero_mayor=cantidad_numero_ingresado

    if aumento == cantidad_numero_ingresado:
        promedio = sumando/cantidad_numero_ingresado
        print("Cantidad Numeros:", cantidad_numero_ingresado)
        print("Suma Total:", sumando)
        print("Promedio:", promedio)
        print("Numero Mayor:", numero_mayor)
        print("Numero Menor:", numero_menor)        
        break